doc/source/statsd.rst

```diff
@@ -25,14 +25,14 @@ All the |metrics| will be created dynamically as the |metrics| are sent to
 `gnocchi-statsd`, and attached with the provided name to the |resource| ID you
 configured.
 
-The `gnocchi-statsd` may be scaled, but trade-offs have to been made due to the
+The `gnocchi-statsd` may be scaled, but trade-offs have to be made due to the
```

Comment:
probably better as "have been" rather than "have to be" as the first part of sentence is past tense.

Reply:
Good catch.